.release-please-manifest.json

```diff
@@ -1,5 +1,5 @@
 {
-  "libs/hooks/open-telemetry": "1.2.3-alpha",
-  "libs/providers/go-feature-flag": "0.1.3",
-  "libs/providers/flagd": "0.3.0"
+  "libs/hooks/open-telemetry": "2.0.0-alpha",
```

Comment:
Looks like the alpha didn't get removed. I feel like there's go to be a configuration somewhere for this...